ibis/tests/all/test_temporal.py

```diff
@@ -244,6 +255,10 @@ def test_temporal_binop(backend, con, alltypes, df, expr_fn, expected_fn):
     result = con.execute(expr)
     expected = backend.default_series_rename(expected)
 
+    # PySpark backend returns diff in days, no support for timedelta
+    if is_timedelta64_dtype(expected.dtype) and isinstance(backend, PySpark):
```

Comment:
If PySpark returns different things, I don't think it should be an supported operation..

Reply:
I was going back and forth on this. I wanted to include date diff because the feature is supported in PySpark, but I can see how it could be confusing if one result type is int and all others is some form of timedelta. I'll remove this and mark date diff and timestamp diff as unsupported.